framework/assets/yii.validation.js

```diff
@@ -328,17 +315,13 @@ yii.validation = (function ($) {
         },
 
         ip: function (value, messages, options) {
-            var getIpVersion = function (value) {
```

Comment:
Separate function is useless here because it's used only once and it's just one short line.